src/client/thirdParty/Amplitude.tsx

```diff
@@ -36,33 +37,51 @@ type AmplitudeProps = {
   apiKey: string;
 };
 
-export const Amplitude: FunctionComponent<AmplitudeProps> = ({ apiKey }) => {
+function initAmplitude(apiKey: string) {
+  amplitude.init(apiKey, {
+    defaultTracking: {
+      attribution: false,
+      sessions: false,
+      formInteractions: false,
+      fileDownloads: false,
+    },
+    serverUrl: AMPLITUDE_INGRESS_PROXY,
+  });
+}
+
+function usePlaygroundSignalsWrapper() {
   usePlaygroundSignals({
     onServerApiSuccess: (event) => {
       const visitorId = event.products.identification?.data?.visitorId;
       const botDetected = event?.products?.botd?.data?.bot?.result === 'bad' ? 'True' : 'False';
 
       amplitude.add(demoPageViewedEventPropertiesEnrichment(botDetected));
-      amplitude.init(apiKey, {
-        defaultTracking: {
-          pageViews: {
-            eventType: EVENT_TYPE,
-          },
-          attribution: false,
-          sessions: false,
-          formInteractions: false,
-          fileDownloads: false,
-        },
-        deviceId: visitorId,
-        serverUrl: AMPLITUDE_INGRESS_PROXY,
-      });
 
-      // Set Amplify user's custom `botDetected` property based on Fingerprint Bot Detection result
-      const identifyEvent = new amplitude.Identify();
-      identifyEvent.set('botDetected', botDetected);
-      amplitude.identify(identifyEvent);
+      amplitude.track(EVENT_TYPE, {
+        botDetected,
+        visitorId,
+      });
     },
   });
+}
+
+export async function trackAskAIkHelpMethodChosen(
```

Comment:
you don't need `async` here, or you should put `return` before `amplitude.track`